src/gui/accountsettings.cpp

```diff
@@ -334,40 +334,43 @@ void AccountSettings::slotEditCurrentIgnoredFiles()
     openIgnoredFilesDialog(f->path());
 }
 
-void AccountSettings::slotOpenMkFolderDialog()
+void AccountSettings::slotOpenMakeFolderDialog()
 {
-    const auto selected = _ui->_folderList->selectionModel()->currentIndex();
+    const QString fileName = [this] () {
```

Comment:
Sorry, probably my fault for being unclear. :-/

What I meant if that the IIL would be nicely used for the part starting now from `QString result`. Using it all the way to selected, classification and friend goes overboard and hinder readability in my opinion. This is pattern which works only for short things where typically the ternary operator wouldn't be enough but not for full blown initialization of many intermediate variables.